AnagramSolver.BusinessLogic/Classes/Repositories/CacheRepository.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AnagramSolver.Contracts.Interfaces;
+using AnagramSolver.EF.DatabaseFirst.Models;
+using System.Threading.Tasks;
+
+namespace AnagramSolver.BusinessLogic.Classes.Repositories
+{
+    public class CacheRepository : ICacheRepository
+    {
+        private readonly VocabularyDBContext _context;
+        public CacheRepository(VocabularyDBContext context)
+        {
+            _context = context;
+        }
+        public async Task<CachedWord> FindCachedWord(string command)
+        {
+            var cachedAnagram = _context.CachedWords.FindAsync(command);
+            return await cachedAnagram;
```

Comment:
Why two lines?

If two lines are kept, variable name is not correct here, because it's not `cachedAnagram`, its `cachedAnagram` Task (`Task<CachedWord>`), which should be named accordingly